guides/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Updated `ActiveSupport::Cache::MemCacheStore` docs to reflect support for `$MEMCACHE_SERVERS`.
+
+    *Sam Bostock*
+
```

Comment:
I'm not sure if this is required. I couldn't find anything describing what changes to the Guides require CHANGELOG entries.